Dockerfile.compute-node

```diff
@@ -869,6 +869,28 @@ RUN case "${PG_VERSION}" in "v17") \
     cargo pgrx install --release && \
     echo "trusted = true" >> /usr/local/pgsql/share/extension/ulid.control
 
+#########################################################################################
+#
+# Layer "pg-session-jwt-build"
+# Compile "pg_session_jwt" extension
+#
+#########################################################################################
+
+FROM rust-extensions-build AS pg-session-jwt-build
+ARG PG_VERSION
+
+RUN case "${PG_VERSION}" in "v17") \
```

Comment:
I think we can try removing this PG 17 hack. This is a Rust extension that does not import any PG symbols, from what I see after a brief look. I'd expect it to work well with any supported version. And `pgrx` seems to support PG17 already

Reply:
You might need to bump the version here, though https://github.com/neondatabase/pg_session_jwt/blob/2a236015f070f875f8308370107374f2fce7b018/Cargo.toml#L21

Looks like PG 17 was added in v0.12.0. The latest release seems to be based on the Postgres 17rc1, so maybe it's not a good idea and better to wait till official release

Reply:
@ololobus we actually tried:

![CleanShot 2024-09-21 at 11 14 30](https://github.com/user-attachments/assets/204bdca3-82cc-4a51-a482-4285bbe96d21)

But we couldn't figure out a way to use 0.12.4 for this extension and 0.11.3 for other extensions. I spent some time on this and eventually had to give up because we need to ship.

I want to put up another PR which bumps all PgRX extensions to 0.12.4 and add PG17 support to all of them.

Reply:
Yeah, makes sense to wait for the full PG 17 release, then figure this out. Sorry for the noise